src/test/java/com/connexta/store/StoreControllerUnitTests.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.store;
+
+import static com.connexta.store.controllers.MultipartFileValidator.MAX_FILE_BYTES;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.when;
+
+import com.connexta.store.controllers.StoreController;
+import com.connexta.store.rest.spring.StoreApi;
+import com.connexta.store.service.api.StoreService;
+import java.io.IOException;
+import javax.validation.ValidationException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.web.multipart.MultipartFile;
+
+// TODO: These test can be changed to test the MultipartFileValidator instead of the controller.
+@ExtendWith(MockitoExtension.class)
+public class StoreControllerUnitTests {
+
+  private static final String STORE_API_VERSION = "testStoreApiVersion";
+
+  @Mock private StoreService mockStoreService;
+
+  private StoreApi storeApi;
+
+  @BeforeEach
+  public void beforeEach() {
+    storeApi = new StoreController(mockStoreService, STORE_API_VERSION);
+  }
+
+  @ParameterizedTest
+  @NullAndEmptySource
+  @ValueSource(strings = "   ")
```

Comment:
❓ Are these parameterized now just so you can add more in the future?

Reply:
No, I also test `NullAndEmptySource`.